src/renderer/components/+extensions/extensions.tsx

```diff
@@ -20,33 +20,47 @@
  */
 
 import "./extensions.scss";
+
 import { remote, shell } from "electron";
 import fse from "fs-extra";
-import { computed, observable, reaction, when } from "mobx";
+import _ from "lodash";
+import { observable, reaction, when } from "mobx";
 import { disposeOnUnmount, observer } from "mobx-react";
 import os from "os";
 import path from "path";
 import React from "react";
-import { autobind, disposer, Disposer, downloadFile, downloadJson, ExtendableDisposer, extractTar, listTarEntries, noop, readFileFromTar } from "../../../common/utils";
-import { docsUrl } from "../../../common/vars";
+import { SemVer } from "semver";
+import URLParse from "url-parse";
+
+import {
+  Disposer,
+  disposer,
+  downloadFile,
+  downloadJson,
+  ExtendableDisposer,
+  extractTar,
+  listTarEntries,
+  noop,
+  readFileFromTar,
+} from "../../../common/utils";
```

Comment:
Is it some auto-formating from vscode?

Reply:
In particular, `TS Hero` plugin 😃

Reply:
Just checked, native "Organize imports" does about the same thing.